lib/jsdom/browser/resources/resource-loader.js

```diff
@@ -103,8 +82,49 @@ module.exports = class ResourceLoader {
 
       case "http":
       case "https": {
-        const requestOptions = this._getRequestOptions(options);
-        return request(urlString, requestOptions);
+        const { accept, cookieJar, referrer } = options;
```

Comment:
It'd be nice if we could put these into the function signature instead of destructuring them here. I need to look at the rest of the function to make sure that works though.